aggregator_core/src/datastore/test_util.rs

```diff
@@ -187,6 +224,19 @@ pub async fn ephemeral_datastore() -> EphemeralDatastore {
     ephemeral_datastore_max_schema_version(i64::MAX).await
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum EphemeralDatastoreError {
```

Comment:
In Janus, test-only code (i.e. literally test code, or code conditionally compiled under the `test-util` feature) is written infallibly, and panics on error (typically via a call to `unwrap` or `expect`).

I think we should get rid of this error type and panic-on-error instead.

Reply:
Indeed. I originally had wanted to ensure the version that failed was captured in the error, but `expect()` with a good message should be just as good.